.github/workflows/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
         stack: [heroku-20, heroku-22, heroku-24]
-        redis_version: ["", "4", "5", "6", "6.2", "7", "7.0", "7.0.11"]
+        redis_version: ["", "4", "5", "6", "6.2", "7", "7.0", "7.0.11", "7.2.5"]
```

Comment:
There is no test coverage of the `7.2` alias.

The idea behind the `redis_version` matrix here is to test:
1. The default version if none specified (ie the empty string case)
2. All permutations of using a version alias (the short form versions, like "4" or "7.2")
3. A single instance of using an exact version, to confirm a specific version can be requested.

ie this should now be something like:

```yaml
        redis_version: ["", "4", "5", "6", "6.2", "7", "7.0", "7.2", "7.0.11"]
```